neurom/fst/_neuronfunc.py

```diff
@@ -140,6 +140,61 @@ def _elevation(section, soma):
             for s in n.neurites if neurite_filter(s)]
 
 
+def trunk_vectors(nrn, neurite_type=NeuriteType.all):
+    '''Calculates the vectors between all the trunks of the neuron
+    and the soma center.
+    '''
+    neurite_filter = is_type(neurite_type)
+    nrns = neuron_population(nrn)
+
+    def _vectors(section, soma):
```

Comment:
don't think you need this helper, just put the morphmath.vector in the list comprehension

Reply:
Ah, yes. Much simpler!